_scripts/cfdoc_linkresolver.py

```diff
@@ -124,7 +124,7 @@ def parseMarkdownForAnchors(file_name, config):
 		# ignore code blocks
 		if line.find("    ") == 0:
 			continue
-		elif line.find("```") == 0:
+		elif line.find("```") == 0 and line.find("```", 3) == -1:
```

Comment:
What's this one?

Reply:
sometimes, line which starts with triple-backtics does not mean start of a code block, like line 18 in [this file][f].

Look at third line in this snippet:
```
You can see the variables and namespace scoped classes defined at the end of an
agent execution by using the ```--show-evaluated-vars``` or
```--show-evaluated-classes``` options to `cf-agent`. In addition to the
variables and classes shown by `cf-promsies --show-classes` or `cf-promises
--show-vars` this will show variables and namespace scoped classes that get
defined during a full agent run where the system may be modified and more policy
is evaluated.
```

[f]: https://github.com/cfengine/documentation/blob/master/guide/faq/show-classes-and-vars.markdown#L18